packages/fhir-cht/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@openfn/language-fhir-cht",
+  "version": "1.0.0",
+  "description": "OpenFn fhir-cht adaptor",
+  "scripts": {
+    "build": "pnpm clean && build-adaptor fhir-cht",
+    "build:src": "pnpm generate-fhir fhir-cht",
+    "build:spec": "pnpm generate-fhir fhir-cht --respec",
+    "test": "mocha --experimental-specifier-resolution=node --no-warnings",
+    "test:watch": "mocha -w --experimental-specifier-resolution=node --no-warnings",
+    "clean": "rimraf dist types docs",
+    "pack": "pnpm pack --pack-destination ../../dist",
+    "lint": "eslint src"
+  },
+  "author": "Open Function Group",
+  "license": "LGPLv3",
+  "type": "module",
+  "fhir": {
```

Comment:
This is a big and noisy PR but I just want to call out this object here

This has really cool metadata about the spec we used to generate the adaptor.

Probably worth noting that both implementations I've worked with seem to have very poor version discipline, so we have to take some of these value with a pinch of salt....